hadoop-ozone/interface-client/pom.xml

```diff
@@ -51,16 +71,46 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
         <extensions>true</extensions>
         <executions>
           <execution>
-            <id>compile-protoc</id>
+            <id>compile-protoc-OmGrpc</id>
             <goals>
               <goal>compile</goal>
               <goal>test-compile</goal>
+              <goal>compile-custom</goal>
+              <goal>test-compile-custom</goal>
             </goals>
             <configuration>
+              <protocArtifact>
+                com.google.protobuf:protoc:${protobuf.version}:exe:${os.detected.classifier}
+              </protocArtifact>
               <protoSourceRoot>${basedir}/src/main/proto/</protoSourceRoot>
+              <includes>
+                <include>OmClientProtocol.proto</include>
+              </includes>
+              <outputDirectory>target/generated-sources/protobuf/java</outputDirectory>
+              <clearOutputDirectory>false</clearOutputDirectory>
+              <pluginId>grpc-java</pluginId>
+              <pluginArtifact>
+                io.grpc:protoc-gen-grpc-java:${grpc-compile.version}:exe:${os.detected.classifier}
```

Comment:
I think this should reference `${io.grpc.version}` for consistency.

Reply:
Thanks @adoroszlai for your comments.  The feature branch was arranged by @elek and I have been working on a local branch submitting PRs to merge into the 'feature branch'.  Note that we have a very large PR, #2259 that we have broken into smaller incremental PRs for submission, starting with this PR #2375.  Ping @elek on updating the feature branch with master - merge `master` to feature branch `HDDS-4440-s3-performance`.

Reply:
Thanks @neils-dev for the info.  I have updated the feature branch after all [CI checks passed](https://github.com/adoroszlai/hadoop-ozone/actions/runs/1021704155) for the same commit in my fork.

Reply:
Thanks @adoroszlai for pointing out the grpc compiler versioning.  Updated for consistency interface-client (ozone) from using `${grpc-compile.version}` with `io.grpc:protock-gen-grpc-java` to `${io.grpc.version}` as suggested.  Following that, removed `grpc-compile.version` from entire build (_pom.xml_) and updated `interface-client `(hdds), `interface-server` (hdds), `ozone-csi ` build files (_pom.xml_) to use `${io.grpc.version}`.

Reply:
Updated with commit with fixes reflecting comments and synched with rebased to Master feature branch.